release/index.md

```diff
@@ -17,7 +17,7 @@ The following table describes the status of Magento software availability and wh
 | **Magento Shipping**                            | Available now                                                                                                                                        | Merchants on Magento 2.2.2+ can use the [onboarding process](https://account.magento.com/shipping/onboarding/start)          |
 | **Page Builder**                                | Available now                                                                                                                                        | Bundled with {{site.data.var.ee}} 2.3.x                                                                                      |
 | **Amazon Sales Channel 2.0.0**                        | Available now for {{site.data.var.ece}} 2.2.4+ and 2.3.x (US, Canada, Mexico)<br><br>EMEA and APAC availability expected in second half of 2019 | Magento Marketplace                                                                                                          |
-| **Google Shopping ads Channel 1.1.2**           | Available now for {{site.data.var.ece}} 2.2.4+ and 2.3.x                                                                                             | [Magento Marketplace](http://marketplace.magento.com/magento-google-shopping-ads.html)                                       |
```

Comment:
One question, is 1.1.2 completely gone? Can someone still have it installed? If not, would keep and add 2.0.0 as a new entry.

Reply:
I've been replacing versions in this table to prevent it from growing to unreasonable sizes. Older versions will still appear in the compatibility table, which I think is sufficient.

Reply:
they can have it already installed.  they can't install an outdated version from marketplace